src/keyboard.rs

```diff
@@ -1577,11 +1588,72 @@ impl Key {
     }
 }
 
+/// The location of the key on the keyboard.
+///
+/// Certain physical keys on the keyboard can have the same value, but are in different locations.
+/// For instance, the Shift key can be on the left or right side of the keyboard, or the number
+/// keys can be above the letters or on the numpad. This enum allows the user to differentiate
+/// them.
+///
+/// See the documentation for the [`location`] field on the [`KeyEvent`] struct for more information.
+///
+/// [`location`]: ../event/struct.KeyEvent.html#structfield.location
+/// [`KeyEvent`]: crate::event::KeyEvent
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum KeyLocation {
+    /// The key is in its "normal" location on the keyboard.
+    ///
+    /// For instance, the "1" key above the "Q" key on a QWERTY keyboard will use this location. This
+    /// invariant is also returned when the location of the key cannot be identified.
+    ///
+    /// ![Standard 1 key](https://i.imgur.com/TGjUpSC.png)
```

Comment:
I'm not sure about using imgur as an image host, given how they've recently decided to [delete "old, unused, and inactive content that is not tied to a user account"](https://arstechnica.com/gadgets/2023/04/hosting-site-imgur-will-remove-explicit-and-anonymous-content-next-month/). I'd prefer to use a light-weight SVG hosted in the repo itself, so that our documentation stays more self-contained.

Reply:
Even after running the photos through `svgo`, each photo is around 450 KB a piece, which would add up to about 2 MB. As a comparison, the total size of the source code is around 2.2 MB in and of itself. Therefore, adding these images to the repo itself would nearly double its size.

If you're fine with that, I'm still willing to do this.

Reply:
I think we can do better than 450 KB if we use lighter SVG as a base, like [this one](https://commons.wikimedia.org/wiki/File:ANSI_US_QWERTY_(Windows).svg), which weights in at 80 KB. It might also be possible to further reduce the size by using SVG links, though I've never used them myself.

Reply:
Yeah, that sounds good. I've changed out for that SVG and include it into the docs.